### DISCUSSION TITLE
Files for localization of RSI Launcher.

### DISCUSSION THREAD

**Discussion by CMDRSnoopy** [Dymerz/StarCitizen-Localization#167] [Q&A] [answered]
Hello, I would like to localize RSI Launcher to my native language. 
There is already a community translation into Star Citizen in the works and I would like to have it complete, if you understand me. The question is if there is any possibility to rewrite RSi Launcher into the appropriate CZECH language.
Thank you in advance for any advice . :)  Adam

**Comment by Auhrus** [comment]
My question would be how can you translate the RSI Launcher?  I only know from the game that you can translate it.🤔

**Reply by CMDRSnoopy** [reply]
I wonder if it's even possible to translate the RSI Launcher, or if it's just a fixed thing that can't be overridden.
Alternatively, I wonder if the file is in the RSI Launcher folder or in the game folder (LIVE)- .pk4

**Comment by ROBdk97** [comment, marked as answer]
It might be challenging to translate the RSI Launcher because it functions more like a web page, using Electron and a Chromium-based web browser. This architecture may limit the ability to translate the app by simply editing files within the application.
